brave/brave-core
# Adds uphold equities card

Resolves https://github.com/brave/brave-browser/issues/12276

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (unit, browser, security tests) on

## Test Plan:
- go to rewards page
- make sure that you see new card at the top

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on.
